portfolio/src/main/webapp/quiz.html

```diff
@@ -18,11 +18,9 @@ <h1 class="header">Music Recommender Quiz</h1>
     </div>
     <div class="quiz">
       <form action="results.html">
-          I want a song that sounds: <br>
-          Sad <input type="range" name="valence" min="0" max="100"> Happy
           <br>
-          I want a song that falls under the following genre: <br>
-          <select name="genre" form="quiz" required="required">
+          <b>I want a song that falls under the following genre: </b>
+          <select class="selectmenu" name="genre" form="quiz" required="required">
```

Comment:
I think for to make it required all you need its `required`. It doesn't need to equal anything.
Separately, does this actually do anything? Like I think by default the first option is selected so it's impossible to make this select empty which would trigger the required warning.